src/components/Product/Product.jsx

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import { useFilter } from "../../Context/filter-context";
+import { useProducts } from "../../Context/products-context";
+import { SorterFun, FilterFun } from "../../sort-filter-funs";
+import Card from "../Card/Card";
+
+export default function Product() {
+  const { prodArr } = useProducts();
+  
+  const {
+    filterstate: { sortby, priceRange, category, condition, rating },
+  } = useFilter();
+
+  const sortedData = SorterFun([...prodArr], sortby);
+  
+  const FilterData = FilterFun(
+    sortedData,
+    priceRange,
+    category,
+    condition,
+    rating
+  );
+  return (
+    <div>
+      <div className="right-side-main">
+        <h1>Showing Products({FilterData.length})</h1>
+
+        <div className="right-side">
+          {FilterData.map((prod) => {
+            return (
+              <Card
+                prodArr={{
+                  prod: prod,
+                }}
```

Comment:
You can change it `prod={prod}` so that you don't need to destructure 2 object in card component